flow360/component/volume_mesh.py

```diff
@@ -415,18 +421,76 @@ def _submit_from_surface(self):
         log.info(f"VolumeMesh successfully submitted: {mesh.short_description()}")
         return mesh
 
+    def _compress_and_upload_chunks(
+        self,
+        upload_id: str,
+        mesh: VolumeMesh,
```

Comment:
Lets make it more generic:
`mesh: VolumeMesh` -> `remote_resource: Flow360ResourceBaseModel`